examples/graphene_apps/python_worker/avalon_worker/worker_hash.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python3
```

Comment:
Lot of code can used reused from sdk functions and common/crypto utility functions. It is good to TODO in the code.

Reply:
`worker_encryption.py`, `worker_signing.py` and `worker_hash.py`  (or similar structure) will be used in Avalon SDK in next release. So as mentioned in ReadMe.md` assumptions, python worker will use Avalon SDK once we have a formal SDK released.  3rd parties developing Graphene SGX worker should be able to build the workers independently without dependency on Avalon repo code.